### DISCUSSION TITLE
Extracting Rich Media such as Video and/or Audio

### DISCUSSION THREAD

**Discussion by manipes-ag** [pymupdf/PyMuPDF#1354] [Looking for help] [answered]
Hi,

Does anybody know how to extract Rich Media from a PDF File? 
**Adobe Acrobat** has support for inserting multimedia files to PDF, link [here](https://helpx.adobe.com/acrobat/using/rich-media.html).

I'm currently scanning through the documentation and only saw the inserting and extraction of embedded files. 

Thank you.

**Comment by JorjMcKie** [comment, marked as answer]
There is currently no support for this - except that you can extract audio content from sound annotations.
But you cannot insert video or audio annotations currently.
